chirp/drivers/uvk5.py

```diff
@@ -0,0 +1,1339 @@
+# Quansheng UV-K5 driver (c) 2023 Jacek Lipkowski <sq5bpf@lipkowski.org>
+#
+# based on template.py Copyright 2012 Dan Smith <dsmith@danplanet.com>
+#
+#
+# This is a preliminary version of a driver for the UV-K5
+# It is based on my reverse engineering effort described here:
+# https://github.com/sq5bpf/uvk5-reverse-engineering
+#
+# Warning: this driver is experimental, it may brick your radio,
+# eat your lunch and mess up your configuration. Before even attempting
+# to use it save a memory image from the radio using k5prog:
+# https://github.com/sq5bpf/k5prog
```

Comment:
Let's remove this warning and recommendation to use the other tool before we merge, please. You can still leave the pointer to the low-level tool, but if we're not confident in this driver's ability to capture a safe backup (and thus the recommendation to use something else for a backup), we shouldn't be merging it.

Reply:
It is safe and,  _now_ that more people are using it, we are confident that the driver works well with these radios.
Removed comment.